gbe/migrations/0022_auto_20221122_2135.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.16 on 2022-11-22 21:35
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('gbe', '0021_auto_20221122_2135'),
+        ('scheduler', '0010_auto_20221122_2135'),
+    ]
```

Comment:
Also - if I were able to reverse, we'd still loose invisible events and default locations - since I decided those features were irrelevant to the new design.  I've decided to just go for it.